Content/InfoLink/infoLink.json

```diff
@@ -29,6 +29,14 @@
   "infoLink_glossary_possibledappcalls": "## Possible dApp Calls\n\nWhen a dApp requests a [pre-authorization](?glossaryAnchor=preauthorizations), you may see in the summary \"possible dApp calls\". This simply indicates that the dApp may add additional actions to your pre-authorization before submitting the transaction to the Radix Network.\n\nNot to worry - you are only approving and signing what you can see in your wallet. Nothing added by the dApp can do anything with your accounts and assets without your knowledge.",
+  "infoLink_glossary_biometricspin": "## Biometrics/PIN\n\nA biometrics/PIN factor is a 16-word seed phrase protected and accessed via your phone's biometrics or PIN security feature. Unlike most crypto wallets, there isn't a single seed phrase controlling all of your stuff – it's just 1 type of factor you can use to protect [Accounts](?glossaryAnchor=accounts) and [Personas](?glossaryAnchor=personas) or use as part of multi-factor [Security Shields](?glossaryAnchor=securityshields).\n\nBecause this is the only kind of factor held on your phone, it won't automatically transfer if you move to a different phone with your [Radix Wallet](?glossaryAnchor=radixwallet). Using a Security Shield for your Accounts and Personas enables easy transfer of wallet control without having to write down and enter a seed phrase. If you create Accounts or Personas with a biometrics/PIN factor, and you don't apply a Security Shield, you'll need to enter the seed phrase to recover access on a different phone.",
```

Comment:
16 is incorrect and invalid 

https://rdxworks.slack.com/archives/C031A0V1A1W/p1734521351634919?thread_ts=1734520365.054799&channel=C031A0V1A1W&message_ts=1734521351.634919